notebook-examples/kge_wikimedia.ipynb

```diff
@@ -0,0 +1,1398 @@
+{
```

Comment:
<p>Is the result topk 10?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/dgl-ke/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Yes the result is top k. I have modified the argument from 5 to 10.</p>